esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -460,7 +485,7 @@ EspHttpBinding::EspHttpBinding(IPropertyTree* tree, const char *bindname, const
         }
     }
 
-    if(m_secmgr.get())
+    if(m_secmgr)
```

Comment:
Valid but unnecessary change. I don't object to the simplified code, but the change only adds overhead (an overloaded operator is called, which in turn calls get). Probably better to limit the PR to actual functional differences.